### PR TITLE
#164677325 Tests should pass even on Travis CI like locally

### PR DESCRIPTION
**Pull request by nkpremices**
#### What does this PR do?
Give the good configurations of Travis 

#### Description of Task to be completed?
Have the tests of the following endpoints passing locally and on Travis with sufficient coverage :

 - POST  api/v1/auth/signup
- POST api/v1/auth/login
- POST api/v1//messages
- GET api/v1/messages

#### How should this be manually tested?
- locally : 
      After cloning the repo, run the command npm cover to see how they pass and which percentage is 
      the coverage.

- Travis : 
     Go to the git repo and see the Travis badge on the readme file.

#### What are the relevant pivotal tracker stories?
#164430194
#164441974
#164441516

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ deploy:
 env:
   global:
     - PORT=3000
-    - NODE_ENV=production
-    - JWT_KEY=secret
+    - NODE_ENV="developement"
+    - JWT_KEY="secret"
```
Comment (ndero) [inline .travis.yml]: Are there not any alternatives to committing your `JWT_KEY` to a public repository?

**Comment by ndero** [comment]
The PR description doesn't match what's in the PR

File: .travis.yml (hunk 1 above)
Comment (premlogic) [reply .travis.yml]: Oh. I don't know if there are. But I have an Idea

Comment (premlogic) [reply .travis.yml]: What if I encrypt it with bcrypt ? 
so that I will be decrypting it when I want to use it in my code?

Comment (ndero) [reply .travis.yml]: I don't think that's a good idea. Let it stay as it is.

Comment (ndero) [reply .travis.yml]: But do research and see if you can add environment variables on Travis outside code.

**Review by ndero — APPROVED** [review]

**Review by ndero — CHANGES_REQUESTED** [review]
Make the PR description match what's in the code.

File: .travis.yml (hunk 1 above)
Comment (premlogic) [reply .travis.yml]: Okay. I'm on it

**Pull request merged**